docs/Build.md

```diff
@@ -66,7 +66,7 @@ In addition to aliasing, the webpack configuration will be as follows:
 | resolve.alias      | Array of aliases | [See section above](#webpack-aliases) |
 | devServer      | Object | [See section below](#webpack-dev-server) |
 
-This configuration will be defined in separate files per build mode - ie one for `development` and one for `production` - to keep the configuration as minimal and readable as possible. Common configuration will be inherited/extended into these files.
+This configuration will be defined in separate files per build mode - ie one for `development` and one for `production` - to keep the configuration as minimal and readable as possible. Common configuration will be inherited/extended into these files. [See this section for more details](#ui-build-implementation).
```

Comment:
To keep the configuration as minimal and readable as possible, configuration will be defined in separate files per build mode - ie one for `development` and one for `production`. These configurations will extend a common configuration file.